src/lib/viewers/image/ImageBaseViewer.js

```diff
@@ -378,6 +378,8 @@ class ImageBaseViewer extends BaseViewer {
                 // click mouse up. In that case reset the image size, mimicking single-click-unzoom.
                 this.zoom('reset');
             }
+
+            this.emitMetric(VIEWER_EVENT.imageZoomClick, this.options.file.id);
```

Comment:
Would it be more useful to send the `in` or `reset` value since file id can be extracted in other ways?